pom.xml

```diff
@@ -89,6 +89,12 @@
         <groupId>junit</groupId>
         <artifactId>junit</artifactId>
         <version>4.12</version>
+        <exclusions>
+          <exclusion>
+            <groupId>org.hamcrest</groupId>
+            <artifactId>hamcrest-core</artifactId>
```

Comment:
That artifact is Hamcrest 2, while `hamcrest-core` and `hamcrest-library` represent Hamcrest 1